packages/workflow-core/src/lib/plugins/external-plugin/api-plugin.ts

```diff
@@ -23,6 +23,7 @@ export class ApiPlugin {
     this.method = pluginParams.method;
     this.headers = {
       'Content-Type': 'application/json',
+      accept: 'application/json',
```

Comment:
can remove

Reply:
It might be beneficial for other API, no downside here